src/page/Registry.js

```diff
@@ -224,11 +224,11 @@ async function renderDir(pathname, entry) {
 
     // no useful files exist in the repository, so opt to attempt
     // showing the contents of the README file instead if it exists.
-    if (files.some(entry => entry.name === "README.md")) {
+    if (files.some(entry => entry.name.toLowerCase() === "readme.md")) {
       const rawUrl =
         entryType === "esm"
-          ? `${entry.url}${path}README.md`
-          : `https://raw.githubusercontent.com/${owner}/${repo}/${entry.branch}/${path}README.md`;
+          ? `${entry.url}${path}${entry.name}`
+          : `https://raw.githubusercontent.com/${owner}/${repo}/${entry.branch}/${path}/${entry.name}`;
```

Comment:
The reason the tests are failing is because the `entry` in `entry.name` here is not referring to the `entry` with with the `README.md`.

You have to store the file entry you find that has a name of README.md and use its name:

```js
    const readme = files.find(
      entry => entry.name.toLowerCase() === "readme.md"
    );
    if (readme) {
      const rawUrl =
        entryType === "esm"
          ? `${entry.url}${path}${readme.name}`
          : `https://raw.githubusercontent.com/${owner}/${repo}/${entry.branch}/${path}${readme.name}`;
```

Reply:
Ahh, thank you very much! Now it passes the tests :)